dockers/docker-fpm-frr/bgpd.conf.j2

```diff
@@ -40,7 +40,7 @@ router bgp {{ DEVICE_METADATA['localhost']['bgp_asn'] }}
   network {{ prefix | ip }}/32
 {% elif prefix | ipv6 and name == 'Loopback0' %}
   address-family ipv6
-    network {{ prefix | ip }}/64
+    network {{ prefix | ip }}/128
```

Comment:
we intend to keep this way. do not change it.

Reply:
@lguohan fixed.

Reply:
not necessary, I am not sure who is using the unified frr configuration.

Reply:
@lguohan ok. Got it.